flow_screen_components/recordDetailFSC/force-app/main/default/lwc/recordDetailFlowScreenComponent/recordDetailFlowScreenComponent.html

```diff
@@ -1,6 +1,9 @@
 <!-- 
```

Comment:
These are some nice enhancements, Hamzas, and I'm glad you're building on your initial contribution. What I'd like you to do is create a draft blog post that describes the new features, with images. If you give me an email address here, I'll invite you as an Author to post on unofficialsf.com. I'll edit the post so don't worry about it being perfect English.  If the feature involves inputs by the user make sure to provide an example of a good input.

Reply:
Thanks, I'll love to write a post.
My Email address : contact@mhamzas.com

Reply:
I've added you as a Contributor to unofficialsf. Go ahead and draft a blog post and let me know when it's ready for review. Thanks!

Reply:
Hi, 

Thanks but did not receive any email yet.

Reply:
I'd also like to post the following on unofficialsf 
https://www.mhamzas.com/blog/2020/07/15/add-to-calenda/

Reply:
> Hi,
> 
> Thanks but did not receive any email yet.

Used forget pass to login